service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -1434,3 +1439,18 @@ func (t *transferQueueActiveTaskExecutor) applyParentClosePolicy(
 		}
 	}
 }
+
+func copyChildWorkflowInfos(
+	input map[int64]*persistencespb.ChildExecutionInfo,
+) map[int64]*persistencespb.ChildExecutionInfo {
+
+	result := make(map[int64]*persistencespb.ChildExecutionInfo)
+	if input == nil {
+		return result
```

Comment:
Why returning empty `map` if input is `nil`? I believe`nil` map should produce `nil` copy.

Reply:
normally child workflow map from workflow is not nil
the nil check exists here in case test will use this function